subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_mqtt

```diff
@@ -3,14 +3,22 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-menuconfig NRF_CLOUD_MQTT
-	bool "nRF Cloud MQTT"
+module = NRF_CLOUD_MQTT
```

Comment:
Just one question as I haven't been able to look all the details of this PR.
Can `CONFIG_NRF_CLOUD_MQTT` and `CONFIG_NRF_CLOUD_REST` still be turned on simultaneously?

Reply:
Yes. That is still possible.